.clang-format

```diff
@@ -18,6 +18,10 @@ IncludeCategories:
     Priority: 310
   - Regex: "Teuchos_*"
     Priority: 320
+  - Regex: "Anasazi_*"
+    Priority: 330
```

Comment:
I think that it would make sens to order the headers from Trilinos alphabetically.